UofT-Frosh-Orientation/orientation-website
# 628 Login Page Background

## **Contribution:**
Changed the images of the login page background to be the updated 2T3 F!rosh Themed (farm).
Organized the image file directory from `login-light-dark` to `client\src\assets\login` and `client\src\assets\darkmode\login` respectively.

### **Issue:**

Closes # [628](https://github.com/UofT-Frosh-Orientation/orientation-website/issues/628)

### **Visuals:** 

![250004525-db78ab48-cd0a-4e7c-a16b-9db65e91d97e](https://github.com/UofT-Frosh-Orientation/orientation-website/assets/56203267/836fee59-0595-4e91-901a-2af760958ae3)
![250004524-b623dcd2-0aae-43c7-a815-563371ff6efa](https://github.com/UofT-Frosh-Orientation/orientation-website/assets/56203267/7572ce75-70a1-4f88-b5ef-188205b62bac)

## **Details:**

### **Testing Steps:**

Steps required to get expected behavior.

- uncomment the login component in pages.jsx
- set readyForFrosh to true
